lib/merchant.rb

```diff
@@ -0,0 +1,11 @@
+class Merchant
+  attr_reader :id,
+              :name
+
```

Comment:
The spec for this project seems somewhat vague, but perhaps we could use an attr_reader for both `@created_at` and `@updated_at`? The reason for this suggestion is that in our CSV files, those times are listed as attributes.